todo_app/run.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn --bind 0.0.0.0:$PORT todo_app.wsgi:app
```

Comment:
Minor but it's always worth adding a shebang (e.g. `#!/bin/bash`) to scripts like this to ensure that they're interpreted correctly and run consistently